lib/chat_api_web/controllers/canned_response_controller.ex

```diff
@@ -0,0 +1,45 @@
+defmodule ChatApiWeb.CannedResponseController do
+  use ChatApiWeb, :controller
+
+  alias ChatApi.CannedResponses
+  alias ChatApi.CannedResponses.CannedResponse
+
+  action_fallback ChatApiWeb.FallbackController
+
+  def index(conn, _params) do
```

Comment:
@jeepers3327 great work on all this! 

main feedback/thoughts:
- for all of these controller methods, make sure we use the `conn.assigns.current_user`'s information to filter the records we're querying
- for example, this `index` method as it is right now would fetch ALL records, rather than just the ones for the authenticated account
- also, for `create` and `update` methods, we need to make sure that the `account_id` always matches the currently authenticated user's account -- we can do this by doing a `Map.merge(params, %{"account_id" => conn.assigns.user.account_id})`, or raising a 403/forbidden error if someone attempts to pass in a different `account_id`

and if you don't mind, it would be great to add typespecs to these methods as well!

(let me know if any of that doesn't make sense or isn't clear)

Reply:
Finish adding filtering for the canned  response controller. I would like to ask if there's a need for another test for the recent code change.

Would like to ask for review or feedback @reichert621 .

Thank you.